dev_tools/conf/.pylintrc

```diff
@@ -7,6 +7,10 @@ score=no
 reports=no
 py-version=3.9
 disable=
+    abstract-class-instantiated,        # TODO: #1440 - enable and fix
+    deprecated-class,                   # TODO: #1440 - enable and fix
+    possibly-used-before-assignment,    # TODO: #1440 - enable and fix
+    used-before-assignment,             # TODO: #1440 - enable and fix
```

Comment:
how did this get here? is this a merge issue?

Reply:
Yeah, I think I've resolved this.